docs/static/security/logstash.asciidoc

```diff
@@ -196,10 +196,11 @@ output {
     Authority's certificate.
 
 [float]
+[role="xpack"]
 [[ls-monitoring-user]]
 ==== Configuring Credentials for Logstash Monitoring
 
-If you plan to ship Logstash {logstash-ref}/monitoring-logstash.html[monitoring]
+If you plan to ship Logstash {logstash-ref}/configuring-logstash.html[monitoring]
```

Comment:
While this does make the link go to a place with info specifically on configuring monitoring, I'm a little condused why a page `configuring-logstash.html` doesn't contain any _other_ information about 🤔 Configuring Logstash.

Reply:
It's part of a much bigger reorg that requires a lot of refactoring, redirects for broken links, etc. What you're seeing here are some interim steps on the way to the bigger picture.